### DISCUSSION TITLE
Regarding forwarding message to bot

### DISCUSSION THREAD

**Discussion by Navpan18** [python-telegram-bot/python-telegram-bot#4007] [Q&A]
Please tell a way so that i can forward the message from a channel/group to telegram bot by python . As it is not directly allowed by telegram api .

**Comment by Poolitzer** [comment]
You need to use a userbot for that.

**Reply by Navpan18** [reply]
Please elaborate your answer

**Reply by Navpan18** [reply]
Actually the bot doesn't have any channel id / message id in which i can forward the message . If that is possible then please share the method

**Reply by Poolitzer** [reply]
Hm, if you add the account with the userbot (a program which uses telegrams free client api to log into an existing normal telegram account and can do stuff ont he accounts behalf) to a channel, you get its id and the message ids of the messages in that channel with every update. And then just forward it to your bot.
